### DISCUSSION TITLE
Orchard Harvest 2023

### DISCUSSION THREAD

**Discussion by viktoriamagyar** [OrchardCMS/OrchardCore#13014] [General]
Thank you for your feedback so far regarding Orchard Harvest. We have received a significant amount of responses so far which helps us to get a better idea of the right place and time. For the location, we are planning to hold the event in a bigger European city with lots of international flights to make it easy for the community to come from most places. I am currently exploring options for the following cities: Lisbon, Barcelona, Frankfurt, or Amsterdam.

The event definitely should take place in Q2. My opinion is that late May before the summer holiday season would be the best option. So we will have enough time to organize, also it is before the summer vacation season.  

I always welcome your opinions and ideas. As we move forward e will keep community members informed of the details.
Also happy new year to everyone! :)

**Comment by urbanit** [comment]
Unfortunately no contacts in these cities. I asked for Cyprus but without luck. cc:/ @viktoriamagyar

**Comment by pepsinio** [comment]
Unfortunately have no contacts in those cities as well. @viktoriamagyar

**Comment by viktoriamagyar** [comment]
@pepsinio @urbanit Thanks for the feedback! Please keep reading my updates and share your opinion about the weekly topics.

**Reply by viktoriamagyar** [reply]
read the following news here: https://github.com/OrchardCMS/OrchardCore/discussions/12724

**Comment by viktoriamagyar** [comment]
Please check out my latest post: https://github.com/OrchardCMS/OrchardCore/discussions/13574
This year we will have a special online Orchard Harvest starting **on the 31st of May at 13:00 UTC!**
